runtime/runtime-params-estimator/src/utils.rs

```diff
@@ -242,6 +252,19 @@ pub(crate) fn is_high_variance(samples: &[f64]) -> bool {
     !all_below_threshold
 }
 
+/// Returns several percentile values from the given vector of costs. For
+/// example, the input 0.9 represents the 90th percentile, which is the largest
+/// gas cost in the vector for which no more than 90% of all values are smaller.
+pub(crate) fn percentiles(mut costs: Vec<GasCost>, percentiles: &[f32]) -> Vec<GasCost> {
```

Comment:
Alternatively `-> impl Iterator<Item=GasCost>` since in most cases we’re later only iterating over the vector.